ac1/_sources/regression_and_line_of_best_fit/outliers.rst

```diff
@@ -26,30 +32,34 @@ increases by 1, the predicted January temperature \__.
 4. Drops by 1 degree
 
 This line fits the data well, and the correlation coefficient between the two
-variables is -0.85, so any predictions are likely to be reliable. But now, make
-this more complicated by adding a data point for Juneau, Alaska, where the
-average January temperature is 31 degrees. For the sake of fun, also suppose
-there was a data entry error and someone entered 331, rather than 31.
+variables is -0.85, so any predictions are likely to be reliable. 
+
+Now compare this to what happens when you add a data point for Juneau, Alaska, 
+where the average January temperature is 31 degrees. Also imagine that there 
+was a data entry error and someone entered 331, rather than 31. Here’s what the 
+graph with the added outlier (the green dot) looks like:
 
 
 .. image:: figures/outlier_jan_temp.png
 
 
-One reason that it’s a really good idea to start with a scatter plot when
-analyzing quantitative data is that it can help identify outliers. If you find a
-data point like the one for Juneau in green above, it’s a good idea to
-investigate. It’s reasonable to guess that there wouldn’t be cities that are so
-unusual and so very far outside the line of best fit. Unusual points like this
-should be investigated and checked if possible.
+Looking at the scatter plot above, it’s easy to identify the outlier because 
+it’s visually far removed from any of the other data points. Outlier 
+identification makes scatter plots a good place to start when analyzing 
```

Comment:
The sentence starting with "Outlier identification" is a bit awkward. Maybe something more like

"Scatter plots are a great place to start when identifying outliers."